mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Neuron.Mixfile do
   use Mix.Project
 
-  @version "0.3.1"
+  @version "0.3.2"
```

Comment:
Please undo this change, I'll do this when bumping the version and generating the changelog.